components/AdminNav/AdminNav.test.tsx

```diff
@@ -1,10 +1,31 @@
 import { AdminNav } from './AdminNav';
+import { AuthContextProvider } from '@/context/AuthContextProvider';
 import { render, screen } from '@testing-library/react';
 import React from 'react';
 
+const mockPush = jest.fn();
+const mockUsePathname = jest.fn();
+
+jest.mock('next/navigation', () => ({
+  useRouter() {
+    return {
+      push: mockPush,
+    };
+  },
+  usePathname() {
+    return mockUsePathname();
+  },
+}));
```

Comment:
Sorry if im wrong, but isn't this only necessary when testing actual navigation? I don't see mockPush being used for now in the tests.

Reply:
@alexappleget line 176.  `expect(mockPush).toHaveBeenCalledWith('/login');`

Reply:
@chris-nowicki the reason I brought it up is because mockPush isn't used anywhere in the AdminNav.test.tsx. What you are referring to is actually in the AdminUserSettings.test.tsx.

Reply:
its confusing though because its needed to pass the test lol.

Reply:
@alexappleget oops just realized this is not my PR! HAHAHA oops.

Reply:
> Sorry if im wrong, but isn't this only necessary when testing actual navigation? I don't see mockPush being used for now in the tests.

I think you may be right. let me take it out